types/ember-data/test/transform.ts

```diff
@@ -15,8 +16,41 @@ class PointTransform extends DS.Transform {
     }
 }
 
+class EnumTransform extends DS.Transform {
+    deserialize<AllowedValue>(
+        serialized: any,
+        options: { allowedValues: Record<string, AllowedValue> },
+    ): AllowedValue {
+        const { allowedValues } = options;
+        const allowedValuesArr = Object.keys(allowedValues).map((key) => allowedValues[key]);
+
+		assert(
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/ember-data/test/transform.ts;
  Line: 27;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.